docs/advanced/telemetry.md

```diff
@@ -140,14 +140,20 @@ Tags:
 
 The `event.name` tag is reported by individual resilience strategies. The built-in strategies report the following events:
 
-- `OnRetry`
-- `OnFallback`
-- `OnHedging`
-- `OnTimeout`
-- `OnCircuitClosed`
-- `OnCircuitOpened`
-- `OnCircuitHalfOpened`
-- `OnRateLimiterRejected`
+| Event                   | Details                                            |
+|-------------------------|----------------------------------------------------|
+| `OnRetry`               | [Link](../strategies/retry.md#telemetry)           |
+| `OnFallback`            | [Link](../strategies/fallback.md#telemetry)        |
+| `OnHedging`             | [Link](../strategies/hedging.md#telemetry)         |
+| `OnTimeout`             | [Link](../strategies/timeout.md#telemetry)         |
+| `OnCircuitClosed`       | [Link](../strategies/circuit-breaker.md#telemetry) |
+| `OnCircuitOpened`       | [Link](../strategies/circuit-breaker.md#telemetry) |
+| `OnCircuitHalfOpened`   | [Link](../strategies/circuit-breaker.md#telemetry) |
+| `OnRateLimiterRejected` | [Link](../strategies/rate-limiter.md#telemetry)    |
+| `Chaos.OnFault`         | [Link](../chaos/fault.md#telemetry)                |
+| `Chaos.OnOutcome`       | [Link](../chaos/outcome.md#telemetry)              |
+| `Chaos.OnLatency`       | [Link](../chaos/latency.md#telemetry)              |
+| `Chaos.OnBehavior`      | [Link](../chaos/behavior.md#telemetry)             |
```

Comment:
Do we need a table here? Could it just be a list where each item is itself a link?